tests/print_status/run.sh

```diff
@@ -40,39 +62,21 @@ function run() {
     cat $status_file|while read -r line; do
         total_file_count=$(echo "$line"|awk '{print $(NF-4)}'|tr -d ",")
         [ $total_file_count -eq 3 ]
-        ((count+=1))
+        count=$((count+1))
         if [ $count -eq $status_count ]; then
             finished_bytes=$(echo "$line"|awk '{print $3}'|tr -d ",")
             total_bytes=$(echo "$line"|awk '{print $6}'|tr -d ",")
             [[ "$finished_bytes" -eq "$total_bytes" ]]
         fi
     done
-
-    run_sql_file $cur/data/db.increment.sql $MYSQL_HOST1 $MYSQL_PORT1
-    check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+    echo "check load unit print status success"
 
     # check sync unit print status
-    i=0
-    while [ $i -lt 3 ]
-    do
-        event_count=$(grep "receive binlog event with header" $WORK_DIR/worker1/log/dm-worker.log|grep -v relay|grep "XIDEvent"|wc -l)
-        if [ "$event_count" == "92" ]; then
```

Comment:
do not check `event_count` anymore?

Reply:
1. before this pr, check `event_count` ensures dm-worker log flushed to disk, it is not used for data replication guarantee (As data has been checked via sync_diff).
2. after this pr, we first check data consistency, kill dm-worker and then check print_status.